packages/cli-kit/src/public/node/output.ts

```diff
@@ -245,20 +245,32 @@ export function collectLog(key: string, content: OutputMessage): void {
 }
 
 export const clearCollectedLogs = (): void => {
-  // console.log('clearCollectLogs')
   collectedLogs = {}
-  // console.log(collectedLogs)
 }
 
 /**
- * Ouputs information to the user.
+ * Outputs command result information to the user.
+ * Result messages don't get additional formatting.
+ * Note: By default, result messages are logged at info level to stdout.
+ *
+ * @param content - The content to be output to the user.
+ * @param logger - The logging function to use to output to the user.
+ */
+export function outputResult(content: OutputMessage, logger: Logger = consoleLog): void {
```

Comment:
It's very weird that we are calling it `outputResult` and yet much of what it outputs isn't results. Do we have a plan to clean this up and move usages over to another function which will be explicitly for non-result stuff to be printed without modification to stderr?

Reply:
It was decided [here](https://github.com/Shopify/cli/pull/4922#discussion_r1927771073). I think it makes sense, we basically want to use this to print the important stuff, usually the "result". What are we printing that you don't think it's a result? Any other suggestion for the name?